Author: Ludovic Brenta <lbrenta@debian.org>
Date: 2012-01-04
Description: Project Editor > Make hangs gnat-gps
 When the user clicks on the Make tab of the Project Editor, GPS hangs.
Index: b/prj_editor/src/project_properties.adb
===================================================================
--- a/prj_editor/src/project_properties.adb
+++ b/prj_editor/src/project_properties.adb
@@ -4920,7 +4920,7 @@
       P                    : Project_Editor_Page;
       Flags                : Selector_Flags;
    begin
-      if Page >= Pages_From_XML_Count
+      if Page > Pages_From_XML_Count
         and then not Gtk.Object.In_Destruction_Is_Set (Ed)
       then
          --  Some pages might not be visible though...
